examples/gallery/line/linestyles.py

```diff
@@ -0,0 +1,39 @@
+"""
+Line styles
+-----------
+
+The :meth:`pygmt.Figure.plot` method can plot lines in different styles.
+The default line style is a 0.25-point wide, black, solid line.
+More line styles can be given via the ``pen`` argument.
```

Comment:
Do you think we should link to the cookbook's advanced `pen` style attributes page at https://docs.generic-mapping-tools.org/latest/cookbook/features.html#wpen-attrib?

Reply:
Yes, I think so. Otherwise, we have to explain what `3p,tomato,4_2:2p` means.